IBAnimatable/PresentationBackgroundView.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  PassthroughBackgroundView.swift
+//  IBAnimatable
+//
+//  Created by Tom Baranes on 28/03/2017.
+//  Copyright © 2017 IBAnimatable. All rights reserved.
+//
+
+import UIKit
+
+final class PresentationBackgroundView: UIView {
```

Comment:
I have no idea... either they do that, otherwise they must have access to a few private API.